Functions/Admin/Backup-RsEncryptionKey.ps1

```diff
@@ -4,71 +4,134 @@
 
-    .PARAMETER SqlServerInstance (optional)
-        Specify the name of the SQL Server Reporting Services Instance.
-
-    .PARAMETER SqlServerVersion (optional)
-        Specify the version of the SQL Server Reporting Services Instance. 13 for SQL Server 2016, 12 for SQL Server 2014, 11 for SQL Server 2012
-
-    .PARAMETER Password
-        Specify the password to be used for backing up the encryption key. This password will be required when restoring the encryption key.
+	    .PARAMETER ReportServerInstance
+            Specify the name of the SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .PARAMETER ReportServerVersion
+            Specify the version of the SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .PARAMETER ComputerName
+            The Report Server to target.
+            Use the "Connect-RsReportServer" function to set/update a default value.
         
-    .PARAMETER KeyPath
-        Specify the path to where the encryption key should be stored. 
+        .PARAMETER Credential
+            The credentials with which to connect to the Report Server.
+            Use the "Connect-RsReportServer" function to set/update a default value.
 
-    .EXAMPLE
-        Backup-RSEncryptionKey -Password 'Enter Your Password' -KeyPath 'C:\ReportingServices\Default.snk'
-        Description
-        -----------
-        This command will back up the encryption key against default instance from SQL Server 2016 Reporting Services
-    
-    .EXAMPLE
-        Backup-RSEncryptionKey -SqlServerInstance 'SQL2012' -SqlServerVersion '11' -Password 'Enter Your Password' -KeyPath 'C:\ReportingServices\Default.snk'
-        Description
-        -----------
-        This command will back up the encryption key against named instance (SQL2012) from SQL Server 2012 Reporting Services 
+	    .PARAMETER Password
+	        Specify the password to be used for backing up the encryption key. This password will be required when restoring the encryption key.
+	        
+	    .PARAMETER KeyPath
+	        Specify the path to where the encryption key should be stored. 
+
+	    .EXAMPLE
+	        Backup-RSEncryptionKey -Password 'Enter Your Password' -KeyPath 'C:\ReportingServices\Default.snk'
+	        Description
+	        -----------
+	        This command will back up the encryption key against default instance from SQL Server 2016 Reporting Services
+	    
+	    .EXAMPLE
+	        Backup-RSEncryptionKey -ReportServerInstance 'SQL2012' -ReportServerVersion '11' -Password 'Enter Your Password' -KeyPath 'C:\ReportingServices\Default.snk'
+	        Description
+	        -----------
+	        This command will back up the encryption key against named instance (SQL2012) from SQL Server 2012 Reporting Services
+	
+		.NOTES
+			Author:      ???
```

Comment:
I'm rather fond of them (and usually add them to every function I create / edit), as it gives me access to a version history when I'm actually out there using the code, rather than in front of git and my editor at home. Insights into what was fixed, should have been fixed and was otherwise broken has often enough helped me fix it on the spot (and later push the fix back to the codebase).
Of course it's still possible to do without, _especially_ when you can already discount this being maintained (which in fact is totally in your own hands here. You _can_ enforce this convention if you wanted to).
That said, I don't see how the CBH clutters the code - I've never had any issues there. CBH and logic are very strictly separated and the notes section isn't displayed unless you use Get-Help -Full, meaning it won't confuse regular users on the field either.

Soo, I've made my case. If the decision remains to remove it, I'll of course comply and remove it in the next update of my PR :)